### PR TITLE
UIOR-1181 Optimize linked instances query to improve performance

### PR DESCRIPTION
**Pull request by usavkov-epam**
## Purpose

https://issues.folio.org/browse/UIOR-1181

Some queries triggered by UI do not use DB indexes during Get Order/Get Po Line/ Open Oder/ Receive order flow. Need to optimize them to improve performance.

```
14:20:12 [352325/orders;830317/orders-storage] [diku] [c88a8880-f8ec-55c5-a808-14cfb0744108] [mod_orders_storage] WARN  CQL2PgJSON           Doing FT search without index for titles.jsonb->>'instanceId', CQL >>> SQL: instanceId = "" >>> titles.jsonb->>'instanceId' ~ '' 
```

## Approach

Delete redundant `instanceId=""`.
## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [ ] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Review by SerhiiNosko — APPROVED** [review]

File: src/components/POLine/LineLinkedInstances/hooks/useLinkedInstances/useLinkedInstances.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ export const useLinkedTitles = line => {
     () => {
       const searchParams = {
         limit: LIMIT_MAX,
-        query: `poLineId=${line.id} and instanceId="" sortby title`,
+        query: `poLineId=${line.id} sortby title`,
```
Comment (zburke) [inline src/components/POLine/LineLinkedInstances/hooks/useLinkedInstances/useLinkedInstances.js:56]: Should `poLineId=${line.id}` also be changed to `poLineId==${line.id}`? It is my understanding from the [CQL docs](https://github.com/folio-org/raml-module-builder/tree/51a5950a83daf5a8aed36526c59225732fcbaad1#cql-relations) that `==` will avoid the substring phrase match for strings.

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,7 @@
 * Add donor info to Order facets. Refs UIOR-1150.
 * Populate claiming interval in POL from Organization record. Refs UIOR-396.
 * Add claiming checkbox to POL details. Refs UIOR-397.
+* Optimize linked instances query to improve performance. Refs UIOR-UIOR-1181.
```
Comment (zburke) [inline CHANGELOG.md:11]: Pedantic: Dedupe.

**Review by zburke — APPROVED** [review]
LGTM, though I think the query can be improved still further.

File: src/components/POLine/LineLinkedInstances/hooks/useLinkedInstances/useLinkedInstances.js (hunk 1 above)
Comment (usavkov-epam) [reply src/components/POLine/LineLinkedInstances/hooks/useLinkedInstances/useLinkedInstances.js:56]: Good point, Zak, thank you!

File: CHANGELOG.md (hunk 2 above)
Comment (usavkov-epam) [reply CHANGELOG.md:11]: Oops, didn't notice, thanks!

**Pull request merged**